app/session.ts

```diff
@@ -1,105 +1,139 @@
-import {
-  app,
-} from 'electron';
-import enhanceWebRequest from 'electron-better-web-request';
+// import { app, OnBeforeSendHeadersListenerDetails } from 'electron';
+// import enhanceWebRequest from 'electron-better-web-request';
 
 // todo(mikael) type correctly the electron-better-web-request public API
 
-const orderListeners = (listeners: any) => {
-  const orderableOrigins = [
-    'ecx-cors', // warning(hugo) minify all keys
-    'bx-dynamic-user-agent',
-    'ecx-api-handler',
-  ];
+// const orderListeners = (listeners: any) => {
+//   const orderableOrigins = [
+//     'ecx-cors', // warning(hugo) minify all keys
+//     'bx-dynamic-user-agent',
+//     'ecx-api-handler',
+//   ];
 
-  const orderedListeners = orderableOrigins.reduce(
-    (orderedList: any[], origin: string) => {
-      const listener = listeners.find(
-        (l: any) => l.context.origin && l.context.origin === origin
-      );
+//   const orderedListeners = orderableOrigins.reduce(
+//     (orderedList: any[], origin: string) => {
+//       const listener = listeners.find(
+//         (l: any) => l.context.origin && l.context.origin === origin
+//       );
 
-      if (listener) {
-        orderedList.push(listener);
+//       if (listener) {
+//         orderedList.push(listener);
 
-        return orderedList;
-      }
+//         return orderedList;
+//       }
 
-      return orderedList;
-    },
-    []
-  );
+//       return orderedList;
+//     },
+//     []
+//   );
 
-  const unorderedListeners = listeners.filter(
-    (l: any) => !orderableOrigins.includes(l.context.origin)
-  );
+//   const unorderedListeners = listeners.filter(
+//     (l: any) => !orderableOrigins.includes(l.context.origin)
+//   );
 
-  return [...unorderedListeners, ...orderedListeners];
-};
+//   return [...unorderedListeners, ...orderedListeners];
+// };
 
-const webRequestHandler = (listeners: any) => {
-  const sortedListeners = orderListeners(listeners);
+// const webRequestHandler = (listeners: any) => {
+//   const sortedListeners = orderListeners(listeners);
 
-  const response = sortedListeners.reduce(
-    async (accumulator: any, element: any) => {
-      if (accumulator.cancel) {
-        return accumulator;
-      }
+//   const response = sortedListeners.reduce(
+//     async (accumulator: any, element: any) => {
+//       if (accumulator.cancel) {
+//         return accumulator;
+//       }
 
-      const result = await element.apply();
+//       const result = await element.apply();
 
-      return { ...accumulator, ...result };
-    },
-    { cancel: false }
-  );
+//       return { ...accumulator, ...result };
+//     },
+//     { cancel: false }
+//   );
 
-  return response;
-};
+//   return response;
+// };
 
-const callbackMethods = [
-  'onBeforeRequest',
-  'onBeforeSendHeaders',
-  'onHeadersReceived',
-];
+// const callbackMethods = [
+//   'onBeforeRequest',
+//   'onBeforeSendHeaders',
+//   'onHeadersReceived',
+// ];
```

Comment:
Could you add a comment explaining why this is commented?
Something like the following should do:

<pre>
Code related to Chrome Extensions, which are (currently) not supported
</pre>

Reply:
It looks like this hack

https://github.com/getstation/desktop-app/blob/f4c2de9c1c7e44cc0e522e7e19a518134f2fa088/app/session.ts#L94-L104

doesn't work anymore.

So I replaced it with [this](https://github.com/viktor44/station-desktop-app/blob/400360863dd1068e6eee484853c271524309308b/app/main.ts#L81-L90)

```javascript
const initWorker = () => {
  app.on('ready', () => {

    session.defaultSession.webRequest.onBeforeSendHeaders((details: OnBeforeSendHeadersListenerDetails, callback: any) => {
      details.requestHeaders['User-Agent'] = getUserAgentForApp(details.url, session.defaultSession.getUserAgent());
      details.referrer = getRefererForApp(details.referrer);
      details.requestHeaders.Referer = details.referrer;

      callback({ cancel: false, requestHeaders: details.requestHeaders });
    });
```